index.html

```diff
@@ -213,15 +221,24 @@ <h2>Scope</h2>
 						Enhancements and improvements to internationalization of technologies defined for EPUB to ensure global usability. While the content of a digital publication (HTML, CSS, etc.) already have advanced internationalization features, an EPUB 3 publication also includes additional textual content like metadata, table of contents, etc., which must abide to all requirements of internationalization. 
 					</li>
 					<li>
-						Integration of new Open Web Platform technologies necessary for enhanced digital publishing experiences, including HTML5 for content documents and new core media types for content.
+						Continued integration of errata and bug fixes.
 					</li>
 					<li>
-						Continued integration of errata and bug fixes.
+						Integration of new Open Web Platform technologies for enhanced digital publishing experiences; these may include:
```

Comment:
To one of Florian's points I saw, should this perhaps be more open-ended, so maybe changed to "these may include but are not limited to:" ?

Reply:
Right. Added it in https://github.com/w3c/epub-3-wg-charter/pull/18/commits/90164e3f06e244d9f54bee76b4bddc12270f839a